platforms/ide/problems/src/main/java/org/gradle/problems/internal/transformers/PluginIdLocationTransformer.java

```diff
@@ -46,7 +46,7 @@ public Problem transform(Problem problem) {
                         problem.getLocations().add(new PluginIdLocation(pluginId));
                     }
                 } catch (Exception ex) {
-                    throw new GradleException("Problem meanwhile reporting problem", ex);
+                    throw new GradleException("Problem while reporting problem", ex);
```

Comment:
The use of `while` here is correct, but I would argue that this error message could be better.  I'm not sure what might make sense to include, but "Problem while reporting problem" seems overly vague.  For example, including identifying information from the `Problem` or the `OperationDetails` might make this quite a bit better.